### PR TITLE
move to MIT license

### LINKED ISSUE #5887: Move to MIT license from Apache
http://github.com/dotnet/roslyn and http://github.com/dotnet/runtime have all to MIT license away from Apache. We should follow suit to simplify code moves between these repositories.

### PR DESCRIPTION
**Pull request by dmonroym**
Fixes #5887

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```
Comment (davkean) [inline Directory.Build.props]: This says "LICENSE" but the license is called "LICENSE.md" - can we make one of them in sync?

Comment (dmonroym) [reply Directory.Build.props]: This is equal to what's on roslyn and runtime where they have License.txt instead, should we change our file from md to txt?

Comment (davkean) [reply Directory.Build.props]: Well they are both wrong too, this is a copy and paste error, it used to be called "LICENSE".

Comment (davkean) [reply Directory.Build.props]: https://github.com/dotnet/project-system/commit/167b0b83c8108df757989f15b03a8db9a795efd4

Comment (dmonroym) [reply Directory.Build.props]: I'll change the file name in the comments then and file an issue with the other repos so they do the same. Just to understand the context, why'd we switch to .md?

Comment (davkean) [reply Directory.Build.props]: Cos it looked better.

Comment (davkean) [reply Directory.Build.props]: @dmonroym I'm fixing this.

**Comment by davkean** [comment]
I've grabbed your commits and thrown them in this PR, fixed the reference to LICENSE.md and enforced it: https://github.com/dotnet/project-system/pull/5965.

**Pull request closed without merging**